src/main/java/uk/gov/hmcts/reform/managecase/client/datastore/CaseDetails.java

```diff
@@ -29,6 +36,34 @@
     private String caseTypeId;
     @JsonProperty("case_data")
     private Map<String, JsonNode> data;
+    @JsonProperty("security_classification")
+    private SecurityClassification securityClassification;
+    @JsonProperty("data_classification")
+    @ApiModelProperty("Same structure as `case_data` with classification (`PUBLIC`, `PRIVATE`, `RESTRICTED`) "
+        + "as field's value.")
+    private Map<String, JsonNode> dataClassification;
+
+    public Optional<String> findChangeOrganisationRequestFieldName() {
+        Optional<JsonNode> first = getData().values().stream()
+            .map(node -> node.findParents(ORGANISATION_TO_ADD))
+            .flatMap(List::stream)
+            .map(node -> node.findParents(ORGANISATION_TO_REMOVE))
+            .flatMap(List::stream)
+            .map(node -> node.findParents(ORGANISATION_CASE_ROLE_ID))
+            .flatMap(List::stream)
+            .map(node -> node.findParents(ORGANISATION_REQUEST_TIMESTAMP))
+            .flatMap(List::stream)
+            .findFirst();
+
```

Comment:
We could use  .findAny(); if it is not mandatory to get only the first. better performance . However it is a minor comments, not need to change it .